tests/utils.go

```diff
@@ -320,43 +320,44 @@ func createDeployVerifyApp() {
 
 func createAndDeployAppPod(appname string) {
 	var err error
-	ginkgo.By("building a busybox app pod deployment using above lvm volume")
+	ginkgo.By("building a fio app pod deployment using above lvm volume")
 	deployObj, err = deploy.NewBuilder().
 		WithName(appname).
 		WithNamespace(OpenEBSNamespace).
 		WithLabelsNew(
 			map[string]string{
-				"app": "busybox",
+				"app": "fio-ci",
 			},
 		).
 		WithSelectorMatchLabelsNew(
 			map[string]string{
-				"app": "busybox",
+				"app": "fio-ci",
 			},
 		).
 		WithPodTemplateSpecBuilder(
 			pts.NewBuilder().
 				WithLabelsNew(
 					map[string]string{
-						"app": "busybox",
+						"app": "fio-ci",
 					},
 				).
 				WithContainerBuilders(
 					container.NewBuilder().
-						WithImage("busybox").
-						WithName("busybox").
+						WithImage("xridge/fio").
+						WithName("fio").
 						WithImagePullPolicy(corev1.PullIfNotPresent).
 						WithCommandNew(
 							[]string{
 								"sh",
 								"-c",
-								"date > /mnt/datadir/date.txt; sync; sleep 5; sync; tail -f /dev/null;",
+								"fio --filename=/mnt/datadir/fioFile --direct=1 --rw=write --bs=4k --ioengine=linuxaio --iodepth=32 --size=3GiB --numjobs=1 --name=fio-ci",
```

Comment:
Do we need to write the entire 3GiB?

Reply:
ok it's not very clear that this is done for a specific test, especially when previous command was just writing to a text file.
Should that kind of test specific behaviour be done on a particular test?

Reply:
Since we are now using `fio` so anyway we have to write _some_ data. How much to write can be made configurable depending on test as I mentioned in one of earlier comment. 
btw 3GiB is completing pretty fast too.